test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -153,11 +167,11 @@ void TestNetworkJobs::testCopyToDir() {
     CPPUNIT_ASSERT(createTestDir());
 
     SyncName filename = Str("testCopyToDir_") + Str2SyncName(CommonUtility::generateRandomStringAlphaNum()) + Str(".txt");
-    CopyToDirectoryJob job(_driveDbId, testFileRemoteId, _dirId, filename);
+    CopyToDirectoryJob job(_driveDbId, testFileRemoteId, _dummyRemoteDirId, filename);
```

Comment:
The method `createTestDir()` already create a temporary directory and store its ID in `_dirID`. In `tearDown`, this temporary directory is automatically deleted.
Do we really need all this `_dummy*` variables?

Reply:
I have removed:
        NodeId _dirId;
And added:
        NodeId _dummyLocalDirId;
        NodeId _dummyRemoteDirId;